diff -up ./src/lightdm.c.nodaemon ./src/lightdm.c
--- ./src/lightdm.c.nodaemon	2013-04-19 02:56:41.000000000 +0000
+++ ./src/lightdm.c	2013-05-01 09:20:57.963353649 +0000
@@ -872,7 +872,7 @@ main (int argc, char **argv)
         g_printerr (/* Text printed out when an unknown command-line argument provided */
                     _("Run '%s --help' to see a full list of available command line options."), argv[0]);
         g_printerr ("\n");
-        return EXIT_FAILURE;
+    /*  return EXIT_FAILURE; */
     }
 
     if (show_version)
