diff --git a/data/lightdm.conf b/data/lightdm.conf
index 693f551..562ed05 100644
--- a/data/lightdm.conf
+++ b/data/lightdm.conf
@@ -19,7 +19,7 @@
 #start-default-seat=true
 #greeter-user=lightdm
 #minimum-display-number=0
-#minimum-vt=7
+minimum-vt=1
 #lock-memory=true
 #user-authority-in-system-dir=false
 #guest-account-script=guest-account
@@ -92,7 +92,7 @@
 #user-session=default
 #allow-guest=true
 #guest-session=UNIMPLEMENTED
-#session-wrapper=lightdm-session
+session-wrapper=/etc/lightdm/Xsession
 #greeter-wrapper=
 #guest-wrapper=
 #display-setup-script=
diff --git a/data/users.conf b/data/users.conf
index 05207ce..4c83c96 100644
--- a/data/users.conf
+++ b/data/users.conf
@@ -11,4 +11,4 @@
 [UserAccounts]
 minimum-uid=500
 hidden-users=nobody nobody4 noaccess
-hidden-shells=/bin/false /usr/sbin/nologin
+hidden-shells=/bin/false /sbin/nologin
