diff -Naur lightdm-1.9.6/data/lightdm.conf lightdm-1.9.6.tpg/data/lightdm.conf
--- lightdm-1.9.6/data/lightdm.conf	2013-10-30 22:34:00.000000000 +0000
+++ lightdm-1.9.6.tpg/data/lightdm.conf	2014-09-12 15:41:51.716731371 +0000
@@ -18,8 +18,8 @@
 [LightDM]
 #start-default-seat=true
 #greeter-user=lightdm
-#minimum-display-number=0
-#minimum-vt=7
+minimum-display-number=0
+minimum-vt=1
 #lock-memory=true
 #user-authority-in-system-dir=false
 #guest-account-script=guest-account
@@ -73,10 +73,10 @@
 [SeatDefaults]
 #type=xlocal
 #xdg-seat=seat0
-#xserver-command=X
+xserver-command=X vt1 -background none
 #xserver-layout=
 #xserver-config=
-#xserver-allow-tcp=false
+xserver-allow-tcp=false
 #xserver-share=true
 #xserver-hostname=
 #xserver-display-number=
@@ -93,10 +93,10 @@
 #user-session=default
 #allow-guest=true
 #guest-session=UNIMPLEMENTED
-#session-wrapper=lightdm-session
+session-wrapper=/etc/lightdm/Xsession
 #greeter-wrapper=
 #guest-wrapper=
-#display-setup-script=
+display-setup-script=/etc/X11/xdm/Xsetup_0
 #display-stopped-script=
 #greeter-setup-script=
 #session-setup-script=
diff -Naur lightdm-1.9.6/data/users.conf lightdm-1.9.6.tpg/data/users.conf
--- lightdm-1.9.6/data/users.conf	2013-07-30 14:09:28.000000000 +0000
+++ lightdm-1.9.6.tpg/data/users.conf	2014-09-12 15:38:47.031603779 +0000
@@ -11,4 +11,4 @@
 [UserAccounts]
 minimum-uid=500
 hidden-users=nobody nobody4 noaccess
-hidden-shells=/bin/false /usr/sbin/nologin
+hidden-shells=/bin/false /sbin/nologin
