diff -Naur lightdm-1.3.3//data/lightdm.conf lightdm-1.3.3.new//data/lightdm.conf
--- lightdm-1.3.3//data/lightdm.conf	2012-08-30 03:25:16.000000000 +0600
+++ lightdm-1.3.3.new//data/lightdm.conf	2012-09-24 14:39:47.520219770 +0600
@@ -26,7 +26,7 @@
 #log-directory=/var/log/lightdm
 #run-directory=/var/run/lightdm
 #cache-directory=/var/cache/lightdm
-#xsessions-directory=/usr/share/xsessions
+xsessions-directory=/etc/X11/dm/Sessions
 #remote-sessions-directory=/usr/share/lightdm/remote-sessions
 #xgreeters-directory=/usr/share/xgreeters
 
@@ -75,7 +75,7 @@
 #user-session=default
 #allow-guest=true
 #guest-session=UNIMPLEMENTED
-#session-wrapper=lightdm-session
+session-wrapper=/etc/X11/Xsession
 #display-setup-script=
 #greeter-setup-script=
 #session-setup-script=
